organizers/views.py

```diff
@@ -346,3 +368,18 @@ def get_current_tabs(self):
 
     def get_queryset(self):
         return models.Application.objects.filter(status=APP_DUBIOUS)
+
+
+class BlacklistApplicationsListView(TabsViewMixin, IsOrganizerMixin, ExportMixin, SingleTableMixin, FilterView):
```

Comment:
This view comes from IsOrganizerMixin, this means that all organizers can acces here. In order to solve this you have to create a new minin on User/mixins.py where the function test_func will return true if the request have permisions or if is direcctor